gr-digital/lib/constellation.cc

```diff
@@ -353,21 +355,16 @@ std::vector<float> constellation::soft_decision_maker(gr_complex sample)
 
 void constellation::max_min_axes()
 {
-    // Find min/max of constellation for both real and imag axes.
-    d_re_min = 1e20;
-    d_im_min = 1e20;
-    d_re_max = -1e20;
-    d_im_max = -1e20;
-    for (size_t i = 0; i < d_constellation.size(); i++) {
-        if (d_constellation[i].real() > d_re_max)
-            d_re_max = d_constellation[i].real();
-        if (d_constellation[i].imag() > d_im_max)
-            d_im_max = d_constellation[i].imag();
-
-        if (d_constellation[i].real() < d_re_min)
-            d_re_min = d_constellation[i].real();
-        if (d_constellation[i].imag() < d_im_min)
-            d_im_min = d_constellation[i].imag();
+    // Find min/max of constellation for both real and imag, axes.
+    d_re_min = std::numeric_limits<decltype(d_re_min)>::max();
+    d_im_min = std::numeric_limits<decltype(d_re_min)>::max();
+    d_re_max = -d_re_min;
```

Comment:
I think it's safer and more correct to use `std::numeric_limits<decltype(d_re_max)>::lowest()`.